org.freecadweb.FreeCAD.appdata.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Has it been submitted upstream?

Reply:
See first comment above!

https://github.com/FreeCAD/FreeCAD/pull/1732